src/device.ts

```diff
@@ -9,6 +9,10 @@ export type DeviceBean = {
   verSw: string;
   name: string;
   dps: DeviceDps;
+  uuid: string;
+  deviceType: number;
+  mac: string;
+  address: string;
```

Comment:
In case it's not always here we should declare it as optional keys.

```suggestion
  uuid?: string;
  deviceType?: number;
  mac?: string;
  address?: string;
```